helm/templates/service.yaml

```diff
@@ -9,5 +9,6 @@ spec:
   ports:
     - port: {{ .Values.service.port }}
       targetPort: {{ .Values.containerPortName }}
+      name: grpc-{{ .Chart.Name }}
```

Comment:
maybe also change this https://github.com/hypertrace/gateway-service/blob/283b78eb77fc73dbbaed1dd6fbaa8df75591059d/helm/values.yaml#L15 to grpc-gateway-service

Reply:
Changed to `containerPortName: grpc`.